apps/entity-service/src/entities/dto/site-query.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { EntityQueryDto } from "./entity-query.dto";
+import { IsOptional } from "class-validator";
+
+export class SiteQueryDto extends EntityQueryDto {
+  @ApiProperty({ required: false })
+  @IsOptional()
+  projectUuid?: string;
```

Comment:
Unfortunately, since these entities all share the same endpoint, this will need to get added to the base EntityQueryDto. The way this is implemented now, there won't be any validation at request time because the controller is only checking against the base EntityQueryDto.

Reply:
moved to EntityQueryDto